src/components/Button/Button.jsx

```diff
@@ -1,8 +1,8 @@
 import "./Button.css";
 
-export const Button = ({buttonType, isDisabled, onClickFunction, children}) => {
+export const Button = ({buttonType, isDisabled=false, onClickFunction, children}) => {
   const handleClick = () => {
-    onClickFunction();
+    onClickFunction && onClickFunction();
```

Comment:
if `onClickFunction` exists, then call it. If we blindly call this we will create a null error